enterprise-suite/tests/prometheus_common

```diff
@@ -0,0 +1,85 @@
+# functions common to smoke_prometheus and smoke_alert
+
+deployments_up() {
+  results=$( kubectl get pods -n $NAMESPACE --no-headers | awk '{print $3;}' | grep -v Running | wc -l )
```

Comment:
NOTE:  This didn't have the `-n $NAMESPACE` bit before.  I think this may have explained some of the flakiness with the original `smoke_prometheus`.  I was having all sorts of flaky behavior with `smoke_alert` until I fixed that.

Reply:
good find!!